osu.Game.Tests/Visual/Online/TestSceneOnlineViewContainer.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using NUnit.Framework;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Online;
+using osu.Game.Online.API;
+using osuTK.Graphics;
+
+namespace osu.Game.Tests.Visual.Online
+{
+    [TestFixture]
+    public class TestSceneOnlineViewContainer : OsuTestScene
+    {
+        private readonly TestOnlineViewContainer onlineView;
+
+        public TestSceneOnlineViewContainer()
+        {
+            Child = new Container
+            {
+                RelativeSizeAxes = Axes.Both,
+                Child = onlineView = new TestOnlineViewContainer()
+            };
+        }
+
+        private class TestOnlineViewContainer : OnlineViewContainer
+        {
+            public new Container<Drawable> Content => base.Content;
+
+            public new LoadingAnimation LoadingAnimation => base.LoadingAnimation;
+
+            public TestOnlineViewContainer()
+                : base(@"Please sign in to view dummy test content")
+            {
+                RelativeSizeAxes = Axes.Both;
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = Color4.Blue.Opacity(0.8f),
+                    },
+                    new OsuSpriteText
+                    {
+                        Text = "dummy online content",
+                        RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                    }
+                };
+            }
+
+            protected override void PopOut() => Content.FadeOut(150, Easing.OutQuint);
```

Comment:
another way to show that it's customizable is by creating a `BufferedContainer` and blur it here.

Reply:
Simply indicating in class XMLDoc that PopIn() and PopOut() can be overridden to customize fade behaviour should be sufficient imo